app/controllers/api/v1/lots_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::LotsController < ApplicationController
+
+  def create
+    product = Product.find_by_id(product_id)
+    lot = Lot.new(product: product)
+    if lot.save
+      render json: lot, status: 201
+    else
+      head 400
+    end
+  end
+
+  private
+
+    def product_id
+      params[:lot][:product_id]
```

Comment:
I think this ultimately has the same effect as strong params, but any reason to not use them?